scalafix-rules/src/main/scala/scalafix/internal/rule/DisableSyntax.scala

```diff
@@ -196,6 +196,12 @@ final class DisableSyntax(config: DisableSyntaxConfig)
               "Default args makes it hard to use methods as functions.",
               m.pos)
           }
+      case Term.ApplyInfix(_, t @ Term.Name("=="), _, _) if config.noUniversalEquality =>
+        Seq(Diagnostic(
+          "noUniversalEquality",
+          "== (universal equality) is disabled",
```

Comment:
What do you think about allowing users to customize the message? I suspect in many codebases the convention is to use an alternative like `===` or similar.

Reply:
Makes sense. I'll get to it tomorrow (running out now)

Reply:
Concretely, we can make this message configurable by adding `noUniversalEqualityMessage: String` with the default message "universal equality should be avoided since it is not typesafe".